warehouse/query-core/src/test/java/datawave/query/jexl/visitors/PushdownUnexecutableNodesVisitorTest.java

```diff
@@ -233,58 +235,58 @@ public PushdownUnexecutableNodesVisitorTest(String testName, String baseQuery, E
         this.expectedPostFieldState = expectedPostFieldState;
         this.expectedFieldIndexPushdown = expectedFieldIndexPushdown;
     }
-    
+
     @Before
-    public void setup() throws TableNotFoundException {
+    public void setup() throws TableNotFoundException, ExecutionException, MarkingFunctions.Exception {
```

Comment:
last two exceptions not needed.  imports not needed either.

Reply:
Removed